gatsby/src/docs/guides/alert-notifications/creating-custom-rules.mdx

```diff
@@ -0,0 +1,118 @@
+
+Event tags are useful for various reasons. First, tags are indexed and this allows you to query your errors in the Sentry `Event Stream` and `Issue Stream` based on specific tag values. In addition, adding the right tags in your code will allow you tell Sentry to notify you only when events with specific details occur.
+
+To create a rule based on a tag's value, select the condition
+
+```
+An event's tags match {key} {match} {value}
+ ```
+
+ A tag-based condition can be used as the sole condition for an alert rule or in conjunction with additional conditions to fine tune the rule:
+
+![Alert Rule Event Tags](alerts_notifications_11.png)
+> In this rule we're leveraging custom tags that our developers added through the Sentry SDK, telling Sentry to notify us via our _#sentry-urgent_ **Slack** channel when _Enterprise_ customers are experiencing errors in critical parts of our application.
+
+### **Event Attributes**
+
+The alert rule system in Sentry is capable of picking out attributes from an event's payload. There are 15 different kinds of attributes that a rule can target. Those include attributes like: `Error Message`, `Error Type`, the `Platform`, `user.id`, `http.method`, `stacktrace.filename`, and others.
+
+To set an attribute-based condition in your rule, select the condition:
+
+ ```
+An event's {attribute} value {match} {value}
+ ```
+
+![Alert Rule Event Attributes 1](alerts_notifications_12.png)
+ > In this example, any event with type “SubscriptionError” indicating an error the billing flow will be routed directly to PagerDuty for our on-call engineer to handle.
+
+In addition, you can also set up rules that account for multiple different attributes at once, and chain that logic together.
+
+![Alert Rule Event Attributes 2](alerts_notifications_13.png)
+> In this more advanced example, a notification containing the message, platform, and type attribute values gets routed to the _Android Dev_ team in _Slack_ if a _RunTimeError_ with the message _Failed to Reload Index_ comes in from our _Java platform_.
+
+### **Event Thresholds**
+
+Often, it’s necessary to create a rule based on a frequency threshold to help determine the significance of an error’s impact and escalation priority. Among other use-cases, threshold conditions can be set in conjunction with tag and attribute based conditions to indicate a spike in errors in a certain environment, release, or page in your app or package in your code.
+
+Sentry provides two threshold conditions based either on the event occurrence or number of affected users:
+
+```
+An issue is seen more than {value} times in {frequency}
+An issue is seen by more than {value} users in {frequency}
+```
+
+![Alert Rule Thresholds](alerts_notifications_10.png)
+
+### **Issue State Changes**
+
+Every Issue in Sentry has a defined state - _Unresolved_, _Resolved_ or _Ignored_. Read about [Issue States](/workflow/notifications/#issue-states) for more information.
+
+#### **1. Regression Alert**
```

Comment:
i dont think we should use numerical prefixes in headers, especially in this kind of case